CHANGES.rst

```diff
@@ -1,5 +1,6 @@
 Changes
 -------
+
```

Comment:
can you add a new line for the next release with this, should probably bump to 2.7.0 since we're removing 3.7 support

Reply:
Done! I didn't know what date to put, so used today as an arbitrary date.